src/AltinnCore/Common/Languages/LanguageController.cs

```diff
@@ -0,0 +1,68 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.IO;
+using IniParser;
+using IniParser.Model;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using AltinnCore.Common.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace AltinnCore.Designer.Controllers
+{
+    public class LanguageController : Controller
+    {
+        private readonly GeneralSettings _generalSettings;
+        public LanguageController(
```

Comment:
Constructor needs a <summary>

Reply:
needs a?

Reply:
summary-tag

Reply:
<summary> added. Indentation here looks weird though. It does not look like this in Visual Studio and "Format Document" does not change anything...

Reply:
also needs a summary for LanguageController method/constructor
and param-tag for the input params (look at how it is done in the CodelistController)

Reply:
Like this? (d22ee15)

Reply:
Perfect 🥇